api/v1beta2/manifest_types.go

```diff
@@ -62,6 +62,10 @@ type ManifestSpec struct {
 	// +nullable
 	// Resource specifies a resource to be watched for state updates
 	Resource *unstructured.Unstructured `json:"resource,omitempty"`
+
+	// ManagedResources specifies a list of resources which are managed
+	// manually by the user. The resources have the following format: <group>/<version>/<kind>.
+	ManagedResources []string `json:"managedResources,omitempty"`
```

Comment:
How about adding `+optional` marker? Maybe it is not strictly required, but we use it already in line 51 for example.

Reply:
I can add it but it is the same as omit empty